servers.models.js

```diff
@@ -14,6 +14,24 @@ exports.getObjByEndpoints = () => {
   });
 };
 
+exports.getCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;
+  `,
+      [article_id]
+    )
+    .then((result) => {
+      const returnComments = result.rows;
+      if (!returnComments.length) {
```

Comment:
We need to also account for an article that exists but there are no comments - this would be a 200 status code and an empty array is returned